.github/workflows/build.yml

```diff
@@ -38,20 +38,20 @@ jobs:
     runs-on: macOS-latest
     strategy:
       matrix:
-        EMULATOR_API: [22]
+        EMULATOR_API: [22, 23, 24, 25, 26, 27]
         EMULATOR_ARCH: [x86_64]
```

Comment:
FYI Google actually [recommends only using x86_64 images for API 26+](https://www.reddit.com/r/androiddev/comments/e5nulf/emulator_29211_and_amd_hypervisor_12_to_stable/f9l6aos?utm_source=share&utm_medium=web2x) 😃 

In my experience using x86_64 images on lower level APIs (21, 22) can lead to slightly worst performance than x86, and I also haven't seen any noticeable improvements using x86_64 on higher API levels.

Reply:
Thanks man I'll test this out. This matrix syntax is so inconvenient though. Since you're here, any idea why we get:

```
Your emulator is out of date, please update by launching Android Studio:
 - Start Android Studio
 - Select menu "Tools > Android > SDK Manager"
 - Click "SDK Tools" tab
 - Check "Android Emulator" checkbox
 - Click "OK"
```

in logs? You call `sdkmanager emulator` so I don't see how this could happen.

Reply:
Yeah this is false information, we always get this when launching emulator from cli.

Reply:
Ah, thanks! I never noticed that.